portfolio/src/main/webapp/background.html

```diff
@@ -3,34 +3,31 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title> Carol's Portfolio </title>
+    <title>Carol's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"> </script>
+    <script src="script.js"></script>
   </head>
   <body>
-    <div class="container">
-      <div class="menu">
-        <ul>
-            <li class="logo"> <a href="index.html"> <img src="images/logo.jpg"> </a> </li>
-            <li> <a href="index.html"> About Me </a> </li>
-            <li class="currentLink"> <a href="background.html" style="color:white"> Background </a> </li>
-            <li> <a href="photos.html"> Photos </a> </li>
-            <li> <a href="contact.html"> Contact Me </a> </li>
-        </ul>
-      </div>
+    <div class="menu">
+      <ul>
+        <li class="logo"><a href="index.html"><img src="images/logo.jpg"></a></li>
+        <li><a href="index.html">About Me</a></li>
+        <li class="currentLink"> <a href="background.html" style="color:white">Background</a></li>
+        <li><a href="photos.html">Photos</a></li>
+        <li><a href="contact.html">Contact Me</a></li>
+      </ul>
     </div>
-    <h1> Background </h1>
+    
+    <h1>Background</h1>
     <p> 
-      I am a computer science major at Caltech, expecting to graduate in June 2022. <br> 
-      So far, some of the theoretical CS classes I have taken are algorithms, decidability and tractability, and learning systems. <br> 
-      Some of the practical CS classes I've taken are data structures, software design, and machine learning. <br> 
+      I am a computer science major at Caltech, expecting to graduate in June 2022.<br> 
```

Comment:
This text is getting long, prefer to break it at 80 characters. 
https://developers.google.com/style/html-formatting#line-length

Reply:
fixed!